kpi/utils/rename_xls_sheet.py

```diff
@@ -48,6 +48,6 @@ def rename_xlsx_sheet(
         raise NoFromSheetError(from_sheet)
     book[from_sheet].title = to_sheet
     stream = BytesIO()
-    writable.save(stream)
+    book.save(stream)
```

Comment:
I would have tried to use same variable names in  `rename_xls_sheet` and `rename_xlsx_sheet` for consistency.